ios/FluentUI/Tab Bar/TabBarItemView.swift

```diff
@@ -248,21 +260,21 @@ class TabBarItemView: UIControl {
 
         if isInPortraitMode {
             container.axis = .vertical
-            container.spacing = Constants.spacingVertical
-            titleLabel.style = .button2
-            titleLabel.maxFontSize = 10
+            container.spacing = tokens.spacingVertical
+            titleLabel.style = tokens.titleLabelPortrait.0
+            titleLabel.maxFontSize = tokens.titleLabelPortrait.1.size
 
             if canResizeImage {
-                suggestImageSize = titleLabel.isHidden ? Constants.portraitImageSize : Constants.portraitImageWithLabelSize
+                suggestImageSize = titleLabel.isHidden ? tokens.portraitImageSize : tokens.portraitImageWithLabelSize
             }
         } else {
             container.axis = .horizontal
-            container.spacing = Constants.spacingHorizontal
-            titleLabel.style = .footnote
-            titleLabel.maxFontSize = 13
+            container.spacing = tokens.spacingHorizontal
+            titleLabel.style = tokens.titleLabelLandscape.0
+            titleLabel.maxFontSize = tokens.titleLabelLandscape.1.size
```

Comment:
I think this should be `titleLabel.font = UIFont.fluent(tokens.titleLabelLandscape, shouldScale: false)`, and then you can simplify the token to just the font info